docs/source/code.rst

```diff
@@ -0,0 +1,59 @@
+Code Guidelines
+-------------------------------------------
+
+Developers of eMach should read this document to understand expectations of code contributed to the eMach codebase. This document outlines the 
+code requirements for ``analyzers`` and ``machine_designs``. Understanding of proper Python coding practices are necessary to contribute to each 
+section of ``eMach``.
+
+Analyzers
+++++++++++++++++++++++++++++++++++++++++++++
+
+Analyzer modules should be located within ``mach_eval/analyzers`` and then placed within the appropriate subdirectory.
+
+An analyzer module may contain multiple analyzers if they are interdependent. Each analyzer must consist of the following classes:
+
+1. At least one Problem class  
+2. One Analyzer class
+3. Results class
```

Comment:
I recommend we order these as:
1. Problem
2. Results
3. Analyzer

(And then update the order of the passages below to match)

Reply:
I disagree here, let's get everyone's input in the meeting tomorrow.